crates/matrix-sdk-ui/src/room_list_service/room.rs

```diff
@@ -154,22 +154,32 @@ impl Room {
     /// contains a local event. Otherwise, it comes from the cache. This method
```

Comment:
nit: comment needs an update here too.

We could also improve the comment by saying that the timeline may contain more info than the cache, since it may have ran some back-pagination (as I was wondering why we would use the latest event from the timeline rather than the cache).

Reply:
Yup!